internal/tmpdir/tmpdir.go

```diff
@@ -15,10 +15,15 @@ var unixTempDirForBigFiles = builtinUnixTempDirForBigFiles
 // DO NOT change this, instead see unixTempDirForBigFiles above.
 const builtinUnixTempDirForBigFiles = "/var/tmp"
 
-// TemporaryDirectoryForBigFiles returns a directory for temporary (big) files.
+// SetTemporaryDirectoryForBigFiles overwrites default temporary directory.
+func SetTemporaryDirectoryForBigFiles(customTmpDir string) {
+	unixTempDirForBigFiles = customTmpDir
+}
+
+// GetTemporaryDirectoryForBigFiles returns a directory for temporary (big) files.
 // On non Windows systems it avoids the use of os.TempDir(), because the default temporary directory usually falls under /tmp
 // which on systemd based systems could be the unsuitable tmpfs filesystem.
-func TemporaryDirectoryForBigFiles() string {
+func GetTemporaryDirectoryForBigFiles() string {
```

Comment:
https://golang.org/doc/effective_go.html#Getters , the `Get` prefix is not conventional in Go.